src/rpm/payload/mod.rs

```diff
@@ -0,0 +1,550 @@
+//! Read/write `newc` (SVR4) format archives.
```

Comment:
I don't mind you vendoring the code, but it's impossible to see the changes you've made on top of the original code. For suggestion only: split the code into two commits; one with the original code, one with your changes on top.

Reply:
Way ahead of you, this is that initial commit, no changes have actually been added yet

Reply:
Perfect 😀

Reply:
Yeah the errors are because of all the dead code